tests/acceptance/test_build.py

```diff
@@ -1251,40 +1214,29 @@ def test_mender_dbus_interface_file(
             "io.mender.Update1.xml",
         ]
 
-        for dbus_enabled in [True, False]:
-
-            # clean up the mender-client
-            build_image(
-                prepared_test_build["build_dir"],
-                prepared_test_build["bitbake_corebase"],
-                bitbake_image,
-                target="-c clean mender-client",
-            )
+        # clean up the mender-client
+        build_image(
+            prepared_test_build["build_dir"],
+            prepared_test_build["bitbake_corebase"],
+            bitbake_image,
+            target="-c clean mender-client",
+        )
```

Comment:
I don't think we need this clean call anymore.

Reply:
Probably not, but I think it's worth having as a safeguard anyway. A bit like doing `rm logfile && echo "first entry" > logfile`.